cypress.config.js

```diff
@@ -29,6 +29,17 @@ module.exports = defineConfig({
           return null;
         },
       });
+
+      on('task', {
+        generateTestData() {
+          return {
+            edited_username: faker.name.firstName(),
+            edited_bio: faker.lorem.word(5),
```

Comment:
Use 'faker.lorem.words(5)' instead of 'faker.lorem.word(5)' to generate 5 random words for the edited_bio.

Reply:
fixed